src/Table/Table.js

```diff
@@ -383,7 +383,8 @@ export default class Table extends React.Component {
 
     // Use scroll table on first render to check if we need to scroll
     // and if content is bigger than its container
-    if (props.useFlex || itemHeight === 0 || itemListHeight > containerHeight) {
+    if ((props.useFlex || itemHeight === 0 || itemListHeight > containerHeight)
+      && props.useScrollTable !== false) {
```

Comment:
So now it defaults to `true` (because we want to maintain the original behavior, where by default it will determine whether or not it needs to render the scrollTable).